### PR TITLE
feat(suite-native): coin price for mainnets

### LINKED ISSUE #8408: Crypto price visible on asset detail
As a user I would like to see the price of crypto on asset detail page in order to be able to make decision whether to buy / sell / receive / send.

Figma: https://www.figma.com/file/Z6AGVUmKQzLNtDozFamW7f/%5Bs2.0%5D-Mobile?type=design&node-id=14455%3A108463&t=JOLVKyyKfmWgzG6I-1

Acceptance criteria:
When user goes on asset detail, he can see the price of related coin

### PR DESCRIPTION
**Pull request by PeKne**
- mainnet accounts show actual price of the coin

## Related Issue

Resolve #8408 

## Screenshots:

https://github.com/trezor/trezor-suite/assets/26143964/caae60d9-d6c5-4cd8-a4eb-0f3cc50bb6a5

### CODE REVIEW

File: suite-common/wallet-core/src/accounts/accountsReducer.ts [hunk 1]
```diff
@@ -184,6 +184,17 @@ export const selectAccountLabel = memoizeWithArgs(
...
 
+export const selectAccountNetworkSymbol = memoizeWithArgs(
+    (state: AccountsRootState, accountKey: AccountKey): NetworkSymbol | null => {
+        const account = selectAccountByKey(state, accountKey);
+
+        if (!account) return null;
+
+        return account.symbol;
+    },
+    { size: 25 },
```
Comment (juriczech) [inline suite-common/wallet-core/src/accounts/accountsReducer.ts]: No need to memoize this as the return will change with each `accountKey` and the memoization is useless.

File: suite-native/module-accounts/src/components/CoinPriceCard.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+                    icon={icon}
+                    iconSize="extraSmall"
+                    size="medium"
+                    variant={badgeVariant}
+                    label={formattedPercentage}
+                />
+            </Box>
+        </Box>
+    );
+};
```
Comment (juriczech) [inline suite-native/module-accounts/src/components/CoinPriceCard.tsx:67]: Isn't this duplication of graph percentage indicator? Please unify into one component and move to atoms if possible

File: suite-native/module-accounts/src/components/CoinPriceCard.tsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    );
+
+    const { currentValue, valuePercentageChange } = useDayCoinPriceChange(
+        networkSymbol,
+        tokenContract,
+    );
+
+    if (!networkSymbol) return null;
+
+    const coinName = token ? token.name : networks[networkSymbol].name;
```
Comment (juriczech) [inline suite-native/module-accounts/src/components/CoinPriceCard.tsx]: Why is token logic in this file when we don't render it for token detail?

File: suite-native/module-accounts/src/hooks/useDayCoinPriceChange.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+import { getFiatRatesForTimestamps } from '@suite-common/fiat-services';
+import { selectFiatCurrencyCode } from '@suite-native/module-settings';
+
+const UNIX_DAY = 24 * 60 * 60;
+
+export const useDayCoinPriceChange = (
+    networkSymbol?: NetworkSymbol | null,
+    tokenAddress?: TokenAddress,
+) => {
+    const [currentValue, setCurrentValue] = useState<number | undefined>(undefined);
```
Comment (juriczech) [inline suite-native/module-accounts/src/hooks/useDayCoinPriceChange.ts]: nit: Prefer using `null` as a default value as `undefined` generally means that variable has not been defined / initialized, however `null` is meant for initializing variables as empty. 

no biggie though 

https://stackoverflow.com/questions/5076944/what-is-the-difference-between-null-and-undefined-in-javascript

File: suite-native/module-accounts/src/hooks/useDayCoinPriceChange.ts [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+            if (!timestampedFiatRates) return;
+
+            const [yesterday, today] = timestampedFiatRates.tickers;
+            setYesterdayValue(yesterday.rates[fiatCurrencyCode]);
+            setCurrentValue(today.rates[fiatCurrencyCode]);
+        };
+
+        getPrices();
+        const refreshInterval = setInterval(getPrices, 30_000);
```
Comment (juriczech) [inline suite-native/module-accounts/src/hooks/useDayCoinPriceChange.ts]: Please no random numbers - create an appropriately named variable

**Review by juriczech — CHANGES_REQUESTED** [review]

File: suite-common/wallet-core/src/accounts/accountsReducer.ts (hunk 1 above)
Comment (PeKne) [reply suite-common/wallet-core/src/accounts/accountsReducer.ts]: Size arg removed, let's discuss usage of memorization for redux selectors later as agreed.  b9ee3bd35

File: suite-native/module-accounts/src/components/CoinPriceCard.tsx (hunk 3 above)
Comment (PeKne) [reply suite-native/module-accounts/src/components/CoinPriceCard.tsx]: token logic removed b9ee3bd35

File: suite-native/module-accounts/src/hooks/useDayCoinPriceChange.ts (hunk 4 above)
Comment (PeKne) [reply suite-native/module-accounts/src/hooks/useDayCoinPriceChange.ts]: changed to `null` b9ee3bd35

File: suite-native/module-accounts/src/hooks/useDayCoinPriceChange.ts (hunk 5 above)
Comment (PeKne) [reply suite-native/module-accounts/src/hooks/useDayCoinPriceChange.ts]: const created in b9ee3bd35

**Review by juriczech — APPROVED** [review]

**Pull request merged**